test/ros_home/rosdep.yaml

```diff
@@ -1,14 +1,19 @@
+---
 testassimp:
   ubuntu:
     apt:
       packages: [assimp-dev]
   debian:
     source:
-      uri: 'https://kforge.ros.org/rosrelease/viewvc/sourcedeps/assimp/assimp-2.0.86-1.rdmanifest'
+      uri:
+        "https://kforge.ros.org/rosrelease/viewvc/sourcedeps/assimp/\
+        assimp-2.0.86-1.rdmanifest"
```

Comment:
I have zero feelings about this.

Would you rather see `# yamllint disable-line rule:line-length` added to each of those lines instead?

Reply:
Or I'd guess I can disable the rule for the whole file. I really don't care - this is just test collateral.

Reply:
I will also only care about this when I"m looking at it and never think about it otherwise. Let's merge as-is (again, I don't want to block) and if I'm motivated to do research and try to find a more aesthetically pleasing line splitting config I'll submit a follow-up.